### PR TITLE
[Merged by Bors] - chore: remove `import Mathlib.Mathport.Rename`

### PR DESCRIPTION
**Pull request by riccardobrasca**
We remove all `import Mathlib.Mathport.Rename`.

This is done using

```bash
find . -name "*.lean" -exec sed -i '/^import Mathlib.Mathport.Rename/d' {} +
```

plus two fixing by hand of files that were missing various imports.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Thanks; looks good!
maintainer merge

**Comment by semorrison** [comment]
bors d+

File: Mathlib/RepresentationTheory/Rep.lean [hunk 1]
```diff
@@ -424,7 +424,6 @@ def MonoidalClosed.linearHomEquiv : (A ⊗ B ⟶ C) ≃ₗ[k] B ⟶ A ⟶[Rep k
   { (ihom.adjunction A).homEquiv _ _ with
     map_add' := fun _ _ => rfl
     map_smul' := fun _ _ => rfl }
-  set_option linter.uppercaseLean3 false in
```
Comment (grunweg) [inline Mathlib/RepresentationTheory/Rep.lean:427]: FWIW, #14914 also includes this: you certainly need not search for all those comments. (Probably, this won't cause a conflict either - well, at least I hope so.)

**Comment by riccardobrasca** [comment]
bors merge

**Pull request closed without merging**